### ISSUE TITLE
[Serious] SQL Error that deleted all inside playlist and brake song upload

### ISSUE THREAD

**Issue by FiloSpaTeam** [AzuraCast/AzuraCast#2906]
**Using Docker installation method**
Yes

**AzuraCast version**
v0.10.2 #1229f2b

**Host Operating System**
Ubuntu 18.04

**Describe the bug**
We tried to upload some songs inside a folder and later assign folder to playlist.

```sql
An exception occurred while executing 'INSERT INTO station_playlist_media (playlist_id, media_id, weight, last_played) VALUES (?, ?, ?, ?)' with params [42, 3403, 32768, 0]: SQLSTATE[22003]: Numeric value out of range: 1264 Out of range value for column 'weight' at row 1
```

**To Reproduce**
Nothing different than upload a songs inside folders. Later assign folders to playlist. During the association folder where disappearing and later we cannot upload anymore.

**Expected behavior**
Playlist loaded with songs

**Comment by SlvrEagle23** [comment]
@FiloSpaTeam How many songs do you have in that playlist? The weight parameter allows for a fairly large number...

**Comment by FiloSpaTeam** [comment]
Over 1000, maybe 1500/1600. We had over 45gb of music before crash.

**Comment by FiloSpaTeam** [comment]
Just to say. Another playlist has over 3800 songs. xD

**Comment by FiloSpaTeam** [comment]
Ok radio is broken again. We just choosen to use our radio as debug machine for this bug. So tell me what you need to catch and i'll help you to understand :)

**Comment by SlvrEagle23** [comment]
@FiloSpaTeam This error should never be causing your station to actually break and stop broadcasting. If anything, it would only affect your ability to assign files to playlists.

I am looking into it and may be able to resolve it shortly, but it really shouldn't be stopping your broadcast entirely.

**Comment by FiloSpaTeam** [comment]
Yes and we lost playlist related to error... But actually is broadcasting... If you need more info please tell.

**Comment by FiloSpaTeam** [comment]
Thanks! I'll check shortly :)
